Packs/Gatewatcher/Integrations/Gcenterv102/Gcenterv102.py

```diff
@@ -0,0 +1,1265 @@
+    """Ignore mac address command.
+
+    Args:
+        client: Client to interact with the GCenter.
+        args: Command arguments.
+
+    Returns:
+        CommandResults object with the 'GCenter.Ignore' prefix.
+    """
+    result = client.ignore_mac_address(
+        mac=args.get("mac"),  # type: ignore
+        start=args.get("start")  # type: ignore
+    )
+    return CommandResults(
+        outputs_prefix="GCenter.Ignore.MacAddress",
+        outputs=result
+    )
+
+
+def gw_del_ignore_asset_name(client: GwClient, args: Optional[Dict[Any, Any]]) -> CommandResults:  # noqa: E501
+    """Delete ignore asset name command.
+
+    Args:
+        client: Client to interact with the GCenter.
+        args: Command arguments.
+
+    Returns:
+        CommandResults object with the 'GCenter.Ignore.AssetName' prefix.
+    """
+    result = client.del_ignore_asset_name(
+        ignore_id=int(args.get("ignore_id"))  # type: ignore
+    )
+    return CommandResults(
+        outputs_prefix="GCenter.Ignore.AssetName",
+        outputs=result
+    )
+
+
+def gw_del_ignore_kuser_ip(client: GwClient, args: Optional[Dict[Any, Any]]) -> CommandResults:  # noqa: E501
+    """Delete ignore Kerberos ip command.
+
+    Args:
+        client: Client to interact with the GCenter.
+        args: Command arguments.
+
+    Returns:
+        CommandResults object with the 'GCenter.Ignore.KuserIP' prefix.
+    """
+    result = client.del_ignore_kuser_ip(
+        ignore_id=int(args.get("ignore_id"))  # type: ignore
+    )
+    return CommandResults(
+        outputs_prefix="GCenter.Ignore.KuserIP",
+        outputs=result
+    )
+
+
+def gw_del_ignore_kuser_name(client: GwClient, args: Optional[Dict[Any, Any]]) -> CommandResults:  # noqa: E501
+    """Delete ignore Kerberos username command.
+
+    Args:
+        client: Client to interact with the GCenter.
+        args: Command arguments.
+
+    Returns:
+        CommandResults object with the 'GCenter.Ignore.KuserName' prefix.
+    """
+    result = client.del_ignore_kuser_name(
+        ignore_id=int(args.get("ignore_id"))  # type: ignore
+    )
+    return CommandResults(
+        outputs_prefix="GCenter.Ignore.KuserName",
+        outputs=result
+    )
+
+
+def gw_del_ignore_mac_address(client: GwClient, args: Optional[Dict[Any, Any]]) -> CommandResults:  # noqa: E501
+    """Delete ignore mac address command.
+
+    Args:
+        client: Client to interact with the GCenter.
+        args: Command arguments.
+
+    Returns:
+        CommandResults object with the 'GCenter.Ignore.MacAddress' prefix.
+    """
+    result = client.del_ignore_mac_address(
+        ignore_id=int(args.get("ignore_id"))  # type: ignore
+    )
+    return CommandResults(
+        outputs_prefix="GCenter.Ignore.MacAddress",
+        outputs=result
+    )
+
+
+def gw_send_malware(client: GwClient, args: Optional[Dict[Any, Any]]) -> CommandResults:  # noqa: E501
+    """Send file to the GScan malcore analysis.
```

Comment:
The difference between malware and malcore is not obvious.
Malcore is the engine on the GCenter appliance that analyses file to detect malware.
Maybe the command is misnamed ? The idea is to find out if a file is a malware.

Reply:
Got it. So maybe it would be worth adding that explanation to the docstring of the command - something like:
"Send a file to the GScan malcore analysis to determine if it is malware".